services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentService.kt

```diff
@@ -72,7 +77,11 @@ class ApplicationAttachmentService(
                 application = application,
             )
 
-        return attachmentRepository.save(applicationAttachment).toMetadata().also {
+        val savedAttachment = attachmentRepository.save(applicationAttachment)
+
+        sendIfPending(application.toApplication(), savedAttachment)
```

Comment:
The API documentation should be updated to say we're uploading the attachment to Allu in some cases.

Reply:
Added.

Reply:
Added.